metal-k8s.yml

```diff
@@ -57,14 +57,10 @@
   gather_facts: False
   roles:
     - role: info_displayer
-      add_info:
-        - master_dashboard_url: >-
-            https://{{ kube_login }}:{{ kube_password }}@{{ ip_master }}:6443/ui
-          kube_login: '{{ kube_login }}'
-          kube_password: '{{ kube_password }}'
       vars:
-        kube_login: 'kube'
-        kube_password: >-
-          {{ lookup('file', 'kubespray/credentials/kube_user') }}
-        ip_master: >-
-          {{ hostvars[groups['kube-master'][0]].ansible_default_ipv4.address }}
+        add_info:
+        - master_dashboard_url: >-
+            https://{{ hostvars[groups['kube-master'][0]].ansible_default_ipv4.address
```

Comment:
Use `external_apiserver_endpoint`, as also used in `kubespray/roles/kubernetes/client/templates/admin.conf.j2`?